packages/ui/src/components/va-stepper/VaStepperControls.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="va-stepper__default-controls">
+    <va-button
+      preset="primary"
+      :disabled="$props.modelValue <= 0"
+      @click="$props.stepControls.prevStep()"
+    >
+      Back
+    </va-button>
+    <va-button
+      v-if="isNextButtonVisible"
+      @click="$props.stepControls.nextStep()"
+      :disabled="$props.nextDisabled"
+    >
+      Next
+    </va-button>
```

Comment:
I wrote in the comment earlier, where I asked to add the `Finish `button by default. What you think?

Reply:
If the component shows "Next" in last step, it will look weird. Let's make people life easier and make finish button.